docs/developers/reference/api/linea-gettransactionexclusionstatusv1.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: linea_getTransactionExclusionStatusV1
+description: Reference content for the linea_getTransactionExclusionStatusV1 method.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_getTransactionExclusionStatusV1`
+
+:::info Only available on request
```

Comment:
I'm able to execute the example request below, so does this mean it's always available using the public endpoint, but if using Infura you need to request access?

Reply:
Also, should users contact Infura support or Linea support.